src/main/java/in/projecteka/consentmanager/ConsentManagerConfiguration.java

```diff
@@ -242,4 +243,9 @@ public JWKSet identityServiceJWKSet(IdentityServiceProperties identityServicePro
     public CentralRegistryTokenVerifier centralRegistryTokenVerifier(@Qualifier("centralRegistryJWKSet") JWKSet jwkSet) {
         return new CentralRegistryTokenVerifier(jwkSet);
     }
+
+    @Bean
+    public CentralRegistryTokenVerifierForGateway centralRegistryTokenVerifierForGateway(@Qualifier("centralRegistryJWKSet") JWKSet jwkSet) {
+        return new CentralRegistryTokenVerifierForGateway(jwkSet);
+    }
```

Comment:
Why do we need this?

Reply:
Bcoz we have other service urls along with gateway calling urls, which doesn't need to check the roles, we added a separate bean so that we do not disturb the existing flow.